craft_archives/repo/projects.py

```diff
@@ -28,6 +30,8 @@
 else:
     KeyIdStr = constr(regex=r"^[0-9A-F]{40}$")
 
+PriorityValue = Optional[Union[Literal["always", "prefer", "defer"], int]]
```

Comment:
What's your opinion on `PriorityValue` being an `Optional` as it is here, versus it being the `Union` bit and then the `priority` attribute being an `Optional[PriorityValue]`? At first glance I think I prefer the second alternative, I think it's more consistent with the other attribute in `Apt`

Reply:
Agreed. Changing